azurerm/internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -171,6 +172,37 @@ func resourceArmMsSqlVirtualMachine() *schema.Resource {
 				ValidateFunc: validate.MsSqlVMLoginUserName,
 			},
 
+			"storage_configuration_settings": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"disk_configuration_type": {
```

Comment:
as is configuration?
```suggestion
						"disk_type": {
```